docs/alaa_review.md

```diff
@@ -0,0 +1,46 @@
+يمكنني أن ألخص أن كراودن (Crowdin) هي جيت هاب (GitHub) غير معقدة و سهلة اللإستعمال لأعمال الترجمة،<br><br>
+</p>
+
+### <p style= "text-align: right">كيف إكتشفت منصة كراودن (Crowdin) لأول مرة <img src="https://media.giphy.com/media/l378xVg7JY3tefx3W/giphy.gif"  height = "30" width = "15"> </p>
+<p style= "text-align: right">
+تمَّ تعريفي على منصة كراودن (Crowdin) أول مرة خلال مساهمتي في عملية ترجمة <a href="https://the-turing-way.netlify.app/welcome">كتاب منهج تورينج (The Turing Way handbook)</a> من اللغة الإنجليزية إلى الغة العربية و هي عملية مدعومة من قبل <a href="https://osc-ksa.com/">مجتمع العلوم المفتوحة في المملكة العربية السعودية (Open Science Community Saudi Arabia</a>،
+<br><br>
+</p>
+
+### <p style= "text-align: right">نبذة عن مجتمع العلوم المفتوحة في المملكة العربية السعودية  <img src="https://raw.githubusercontent.com/Open-Science-Community-Saudi-Arabia/localisation/main/imgs/logo.png" height = "80"  style = "vertical-align: middle"></p>
+<p style= "text-align: right">
+مجتمع العلوم المفتوحة في المملكة العربية السعودية تم إنشاؤه تماشيا مع رؤية المملكة العربية السعودية لعام 2030، والتي تُركِز على تثبيت القيم و تعزيز المعرفة وتحسين تكافؤ فرص الحصول على التعليم، <br> يهدف مجتمع العلوم المفتوحة في المملكة العربية السعودية إلى توفير مكان يتفاعل فيه القادمين الجدد والأقران ذوي الخبرة، وإلهام بعضهم البعض لتمكين الممارسات العلمية المفتوحة والقيم في سير العمل لديهم وتعزيز السياسات التي تدعم توفير بُنية تحتية للعلوم المفتوحة، مجتمع العلوم المفتوحة في المملكة العربية السعودية هو جزء من الشبكة الدولية لمجتمعات العلوم المفتوحة والمكونة من أربعة عشر مجتمع في أوروبا،<br><br>
+</p>
+
+### <p style= "text-align: right"> نبذة عن منهج تورينج (The Turing Way) <img src="https://raw.githubusercontent.com/alan-turing-institute/the-turing-way/main/book/website/_build/html/_static/logo.jpg" height = "100"  style = "vertical-align: middle"></p>
+<p style= "text-align: right">
+منهج تورينج (The Turing Way) هو مشروع مفتوح المصدر، مفتوح للتعاون و مُدار من قبل مجتمَعه، يتضمن و يدعم مجتمعا متنوعا من المساهمين لجعل علوم البيانات متاحة، مفهومة و فعالة للجميع، <br> يهدف مشروع منهج تورينج لتوفير كل المعلومات التي يحتاجها الباحثين و علماء البيانات من القطاع الأكاديمي أو الصناعي أو العام لضمان سهولة جعل البحوث التي يعملون عليها قابلة لإعادة الإنتاج و الإستعمال،<br>
+**ملاحظة:** بالنسبة لشخص ليس له إلا دراية سطحية بعلم البيانات و مجال البحث فإني من خلال قرائتي جزئا من كتاب منهج تورينج إكتشفت و تعلمت أشياءا لم أكن أدريها من قبل، إنَّ بنية الكتاب و طريقة شرحه غير معقدة و مناسبة للمبتدئين،<br>
+</p>
+
+### <p style= "text-align: right"> دليل المبتدئين في القيام بعمليات الترجمة بإستعمال منصة كراودن (Crowdin) </p>
+* <p style = "text-align: right">بعد أن تنشأ حسابا في كراودن استعمل الرابط الذي سيتم مشاركته معك من قبل أحدالمنظمين في المشروع الذي ستساهم في ترجمته و ستتم إضافة المشروع إلى قائمة مشاريعك في كراودن، بعد هذا يمكنك الولوج و ستبدو الواجهة مشابهة لهذه</p><img src= "https://user-images.githubusercontent.com/100871135/164353822-b72ef680-53a4-42c3-9f87-a47ef4cc8a4b.PNG">
```

Comment:
The alignment from left to right n this text and the one that follows

Reply:
@BatoolMM  I made changes to the file , does the problem remains ?